docs/elasticsearch-specification.asciidoc

```diff
@@ -484,13 +487,13 @@ spec:
       maxUnavailable: 1
 ----
 
-`maxSurge` is unbounded: this means that all the missing Pods are immediately created according to the best practices at any given time.
-`maxUnavailable` defaults to `1`: this assures that clusters have at most one unavailable Pod at any given time.
+`maxSurge` is unbounded: This means that all the missing Pods are immediately created according to the best practices at any given time.
+`maxUnavailable` defaults to `1`: This assures that clusters have at most one unavailable Pod at any given time.
 
 ==== Caveats
 * With both `maxSurge` and `maxUnavailable` set to `0`, the operator cannot bring down an existing Pod nor create a new Pod.
-* Due to the safety measures employed by the operator, a particular changeBudget might be insufficient to perform the requested update. For example, with `maxSurge` set to 0 you cannot remove the last data node from one nodeSet and add a data node to a different nodeSet. In this case the operator cannot remove the old node and migrate data as other nodes are not available, and can't add the new node because `maxSurge` is set to 0.
-* Currently, the operator has a limited understanding of the cluster topology and during the update might not pick the optimal order of Pods being added or removed. This can block the update process and you might need to specify a higher `maxSurge` than theoretically needed.
+* Due to the safety measures employed by the operator, a particular `changeBudget` might be insufficient to perform the requested update. For example, with `maxSurge` set to 0 you cannot remove the last data node from one `nodeSet` and add a data node to a different `nodeSet`. In this case, the operator cannot remove the old node and migrate data as other nodes are not available, and can't add the new node because `maxSurge` is set to 0.
```

Comment:
```suggestion
* Due to the safety measures employed by the operator, certain `changeBudget`s might prevent the operator from making any progress . For example, with `maxSurge` set to 0, you cannot remove the last data node from one `nodeSet` and add a data node to a different `nodeSet`. In this case, the operator cannot create the new node because `maxSurge` is 0, and it cannot remove the old node because there are no other data nodes to migrate the data to.
```